src/SQLiteDatabaseIntegrationLoader.php

```diff
@@ -62,13 +62,13 @@ protected static function get_plugin_directory() {
 	 * @return void
 	 * @throws WP_CLI\ExitException
 	 */
-	protected function load_dependencies() {
+	public static function load_plugin() {
```

Comment:
After discussing the initial changes in this PR with @wojtekn, we concluded that this method could also be static, and there's really no reason for the command classes to extend this class. Hence, I took the opportunity to make this into a standalone class with only static methods.